seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/config/SinkConfig.java

```diff
@@ -0,0 +1,192 @@
+                    "The way to specify the parameter is to add the prefix `sink.properties.` to the original stream load parameter name ");
+
+    public static final Option<Integer> BATCH_MAX_SIZE = Options.key("batch_max_rows")
+            .intType()
+            .defaultValue(DEFAULT_BATCH_MAX_SIZE)
+            .withDescription("For batch writing, when the number of buffers reaches the number of batch_max_rows or the byte size of batch_max_bytes or the time reaches batch_interval_ms, the data will be flushed into the Doris");
+
+    public static final Option<Long> BATCH_MAX_BYTES = Options.key("batch_max_bytes")
+            .longType()
+            .defaultValue(DEFAULT_BATCH_BYTES)
+            .withDescription("For batch writing, when the number of buffers reaches the number of batch_max_rows or the byte size of batch_max_bytes or the time reaches batch_interval_ms, the data will be flushed into the Doris");
+
+    public static final Option<Integer> BATCH_INTERVAL_MS = Options.key("batch_interval_ms")
+            .intType()
+            .noDefaultValue()
+            .withDescription("For batch writing, when the number of buffers reaches the number of batch_max_rows or the byte size of batch_max_bytes or the time reaches batch_interval_ms, the data will be flushed into the Doris");
+
+    public static final Option<Integer> MAX_RETRIES = Options.key("max_retries")
+            .intType()
+            .noDefaultValue()
+            .withDescription("The number of retries to flush failed");
+
+    public static final Option<Integer> RETRY_BACKOFF_MULTIPLIER_MS = Options.key("retry_backoff_multiplier_ms")
+            .intType()
+            .noDefaultValue()
+            .withDescription("Using as a multiplier for generating the next delay for backoff");
+
+    public static final Option<Integer> MAX_RETRY_BACKOFF_MS = Options.key("max_retry_backoff_ms")
+            .intType()
+            .noDefaultValue()
+            .withDescription("The amount of time to wait before attempting to retry a request to Doris");
+
+    public enum StreamLoadFormat {
+        CSV, JSON;
+        public static StreamLoadFormat parse(String format) {
+            if (StreamLoadFormat.JSON.name().equals(format)) {
+                return JSON;
+            }
+            return CSV;
+        }
+    }
+
+    private List<String> nodeUrls;
+    private String username;
+    private String password;
+    private String database;
+    private String table;
+    private String labelPrefix;
+    private String columnSeparator;
+    private StreamLoadFormat loadFormat = DEFAULT_LOAD_FORMAT;
+
+    private int batchMaxSize = DEFAULT_BATCH_MAX_SIZE;
+    private long batchMaxBytes = DEFAULT_BATCH_BYTES;
+
+    private Integer batchIntervalMs;
+    private int maxRetries;
+    private int retryBackoffMultiplierMs;
+    private int maxRetryBackoffMs;
+
+    private final Map<String, Object> streamLoadProps = new HashMap<>();
```

Comment:
```suggestion
    private final Map<String, String> streamLoadProps = new HashMap<>();
```

Reply:
![e03a317fc129d3d3d189d0eb81408c3](https://user-images.githubusercontent.com/45089228/205657797-93a14a87-8d7e-4fa5-b753-0d07538b6702.png)

Need to put  Object type.

Reply:
toString()

Reply:
Done.Thanks for your advice.